error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/DirectReturn.java

```diff
@@ -91,12 +97,26 @@ public Description matchBlock(BlockTree tree, VisitorState state) {
                     SuggestedFix.builder()
                         .replace(
                             precedingStatement,
-                            String.format("return %s;", SourceCode.treeToString(resultExpr, state)))
-                        .delete(finalStatement)
+//                            String.format("return %s;", SourceCode.treeToString(resultExpr, state)))
+                            String.format("return %s;", SourceCode.treeToString(resultExpr)))
+                            //Same case for trying to delete wrongly assumed "source code".
+                            //Note 2: Seems we can only use ONE fix because of the way the errorprone replaces "source code"
```

Comment:
```suggestion
                            //Note 2: Seems we can only use ONE fix because of the way the errorprone replaces "source code" where we have two replacements in the same start/end position causing the "conflicting replacement" exception.
```